src/Server/PostgreSQLHandler.cpp

```diff
@@ -172,7 +177,17 @@ void PostgreSQLHandler::run()
                     LOG_DEBUG(log, "Client closed the connection");
                     return;
                 case PostgreSQLProtocol::Messaging::FrontMessageType::PARSE:
+                    processParseQuery();
+                    message_transport->flush();
+                    break;
                 case PostgreSQLProtocol::Messaging::FrontMessageType::BIND:
+                    processBindQuery();
+                    message_transport->flush();
+                    break;
+                case PostgreSQLProtocol::Messaging::FrontMessageType::EXECUTE:
+                    processExecuteQuery();
+                    message_transport->flush();
+                    break;
                 case PostgreSQLProtocol::Messaging::FrontMessageType::DESCRIBE:
                 case PostgreSQLProtocol::Messaging::FrontMessageType::SYNC:
```

Comment:
`DESCRIBE` and `SYNC` also should be processed appropriately
https://www.postgresql.org/docs/current/protocol-flow.html#PROTOCOL-FLOW-EXT-QUERY